src/components/assistant-code-block.tsx

```diff
@@ -50,6 +52,12 @@ const LanguageBlock = ( props: ContextProps & CodeBlockProps ) => {
 		}
 	}, [ blocks, cliOutput, content, setCliOutput, setCliStatus, setCliTime ] );
 
+	const { terminalWpCliEnabled } = useFeatureFlags();
+	const { selectedSite } = useSiteDetails();
+	if ( ! selectedSite ) {
+		return null;
+	}
```

Comment:
This is likely an edge case as `selectedSite` should always be defined in the Assistant tab, but since it's only used in the `onCopied` handler, I wonder if we should only disable that action when `selectedSite` is not defined. WDYT?

Reply:
Done https://github.com/Automattic/studio/pull/563/commits/53adacfeec3bfe563b6e8783b416f6abeae83791